### DISCUSSION TITLE
Build error after update Eclipse

### DISCUSSION THREAD

**Discussion by sugarnet** [eclipse-m2e/m2e-core#1791] [General]
Hello everyone!
I have the following scenario, I just updated Eclipse to version 2024-06 (4.32.0) Build id: 20240606-1231. The update was successful. On the other hand I have Maven configured to point to a different repository than the central maven. The problem with this is that when it tries to build the projects it looks for dependencies (such as maven-surefire-plugin) in my company's repository instead of maven central.
The dependencies that I have configured in my project's pom are all in my company's repository, but when I do: right click->Maven->Update Project->OK I get an error similar to the following:

org.apache.maven.plugins:maven-surefire-plugin:jar:3.2.5 was not found in maven-dev.dscifo/repository during a previous attempt.

My project does not need this jar, but Eclipse obviously does and tries to look for it in the company's private repository instead of maven central.

The question is, can I configure Eclipse to avoid this error?

**Comment by laeubi** [comment]
This is a maven error, so it depends on how exactly you have configured this? Can you probably came up with a small reproducer project?

**Comment by sugarnet** [comment]
Thanks @laeubi . I resolved the problem.
